node/src/actors/messages.rs

```diff
@@ -253,39 +253,37 @@ impl Message for GetUtxoInfo {
     type Result = Result<UtxoInfo, failure::Error>;
 }
 
+/// Reputation info
+#[derive(Clone, Debug, Default, Eq, PartialEq, Serialize, Deserialize)]
+pub struct ReputationStats {
+    /// Reputation
+    pub reputation: Reputation,
+    /// Eligibility
```

Comment:
I would be good to add a short explanation on what's the difference between eligibility and reputation, or link to the function that calculates eligibility given reputation.